src/with-next-video.ts

```diff
@@ -59,7 +59,7 @@ export async function withNextVideo(nextConfig: any, videoConfig: VideoConfigCom
 
       config.experiments.buildHttp = {
         allowedUris: [
-          /https?:\/\/.*\.(mp4|webm|mkv|ogg|ogv|wmv|avi|mov|flv|m4v|3gp)$/,
+          /https?:\/\/.*\.(mp4|webm|mkv|ogg|ogv|wmv|avi|mov|flv|m4v|3gp)\??(?:&?[^=&]*=[^=&]*)*$/,
```

Comment:
What does this add?

Reply:
it adds support for query parameters in the imported source URL.
for remote files it's possible it's required for access control or something else,... Dropbox dl=1 for example